types/global.d.ts

```diff
@@ -0,0 +1 @@
+export type Lang = 'cn' | 'de' | 'en' | 'fr' | 'ja' | 'ko';
```

Comment:
I think to follow an order like `'en' | 'de' | 'fr' | 'ja' | 'cn' | 'ko'` is consistency with our [common locale order](https://github.com/quisquous/cactbot/blob/main/docs/RaidbossGuide.md#trigger-properties).